services/app-web/src/common-code/proto/healthPlanFormDataProto/toDomain.ts

```diff
@@ -320,6 +356,7 @@ const toDomain = (
 
     // Protos support multiple rate infos for now, but we only support one in our domain models
     // so if there are multiple we'll drop the extras.
+    // This should be removed once all multi-rate UI features have been implemented. We should then only use rate data from rateInfos list of rate data.
```

Comment:
When would it be removed? I'm confused why not now. Or which feature are you thinking this should be shipped with?

Reply:
I was thinking in [MR-2527](https://qmacbis.atlassian.net/browse/MR-2527). But I can do that now and leave UI work for MR-2527. I think this will mean it's blocked by this PR.